ui5/browser/controller/Browser.controller.js

```diff
@@ -691,133 +697,96 @@ sap.ui.define(['sap/ui/core/mvc/Controller',
             rows[k].$().dblclick(this.onRowDblClick.bind(this, rows[k]));
          }
       },
 
       /** @brief Send RBrowserRequest to the browser */
-      sendBrowserRequest: function(_oper, args) {
-         var req = { path: "", first: 0, number: 0, sort: _oper };
+      sendBrowserRequest: function (_oper, args) {
+         var req = {path: "", first: 0, number: 0, sort: _oper};
          JSROOT.extend(req, args);
          this.websocket.Send("BRREQ:" + JSON.stringify(req));
       },
 
-     sendDblClick: function(fullpath, opt) {
-        this.websocket.Send('DBLCLK: ["'  + fullpath + '","' + (opt || "") + '"]' );
-     },
+      sendDblClick: function (fullpath, opt) {
+         this.websocket.Send('DBLCLK: ["' + fullpath + '","' + (opt || "") + '"]');
+      },
 
       /** @brief Double-click event handler */
-      onRowDblClick: function(row) {
-        var ctxt = row.getBindingContext(),
+      onRowDblClick: function (row) {
+         let ctxt = row.getBindingContext(),
             prop = ctxt ? ctxt.getProperty(ctxt.getPath()) : null,
-            fullpath = (prop && prop.fullpath) ? prop.fullpath.substr(1, prop.fullpath.length-2) : "";
-
-        if (row._bHasChildren) {
-          let rowText = row.getCells()[0].getContent()[1].getText().substr(1);
-          if(!rowText.endsWith(".root")) {
-            let oBreadcrumbs = this.getView().byId("breadcrumbs");
-            let links = oBreadcrumbs.getLinks();
-            let currentText =  oBreadcrumbs.getCurrentLocationText();
-            let path = "/";
-            for (let i = 1; i<links.length; i++) {
-              path += links[i].getText() + "/";
-            }
-            path += currentText + "/";
-
-            if (row._iLevel !== 0) { // If the clicked row is a child, i need to find all the path from that child to the upper parent
-              let ilevel = row._iLevel;
-              let rows = row.getParent().getRows();
-              let rowIndex;
-              let result = [];
-              let i;
-              for (i=0; i<rows.length; i++) {
-                if (rows[i] === row) {
-                  rowIndex = i;
-                  break;
-                }
-              }
-              for (i = rowIndex; i !== -1; i--) {
-                if (rows[i]._iLevel === ilevel-1) {
-                  result.push(rows[i].getCells()[0].getContent()[1].getText().substr(1));
-                  ilevel--;
-                  if(ilevel === 0) {
-                    break;
-                  }
-                }
-              }
-              result = result.reverse();
-              result.push(rowText);
-              for (i=0; i<result.length; i++) {
-                path += result[i] + "/";
-              }
-            } else {
-              path += rowText + "/";
+            fullpath = (prop && prop.fullpath) ? prop.fullpath.substr(1, prop.fullpath.length - 2) : "";
+
+         if (!fullpath) return;
+
+         if (row._bHasChildren) {
```

Comment:
Can one access `_bHasChildren` via openui5 method?
Can you check this?

Reply:
Ok, will check !

Reply:
Unfortunately no, i did not find any function that return that informations, i have checked in the documentation, and in the code of UI5